pkg/plugin/work-in-progress/event_handler.go

```diff
@@ -163,7 +163,7 @@ func (gh *GitHubWIPPRHandler) hasWorkInProgressLabel(labels []*gogh.Label, wipLa
 }
 
 // HasWorkInProgressPrefix checks if title is marked as Work In Progress
-func (gh *GitHubWIPPRHandler) HasWorkInProgressPrefix(title string, config PluginConfiguration) (bool, string) {
+func HasWorkInProgressPrefix(title string, config PluginConfiguration) (bool, string) {
```

Comment:
I would make this func definition as `func HasWorkInProgressPrefix(title string, config PluginConfiguration) (string, bool)`, so even if you can use `prefix, ok := wip.HasWorkInProgressPrefix(*event.PullRequest.Title, wip.LoadConfiguration(log, change))`. current use of this function looks little weired. - https://github.com/arquillian/ike-prow-plugins/pull/169/files#diff-4228ddbb4e17d1865d8e954407a5fa84R71

Reply:
The way how are we using as referred in above comment - https://github.com/arquillian/ike-prow-plugins/pull/169/files#diff-4228ddbb4e17d1865d8e954407a5fa84R71, like
`ok, prefix := wip.HasWorkInProgressPrefix(*event.PullRequest.Title, wip.LoadConfiguration(log, change));` is somehow different than go way. Because I would expect `ok` to be as second parameter. I am fine with it if we rename `ok` to some meaningful variable like `prefixed`

Reply:
>  is somehow different than go way. Because I would expect ok to be as second parameter.

this would be valid if the method had a name `GetWorkInProgressPrefix` or `SeparateWorkInProgressPrefix`. However, as we were focused on checking the existence of the prefix from the very beginning and getting the separated prefix is just a side effect, then having it in the current way makes sense. 
So, to solve this situation - how about renaming it to `GetWorkInProgressPrefix` with the return values `(string, bool)`...? Does it make sense to both of you @dipak-pawar @hemanik ?

Reply:
@MatousJobanek +1 for your suggestion